renderdoc/core/replay_proxy.cpp

```diff
@@ -1707,6 +1707,21 @@ bool ReplayProxy::SendReplayCommand(ReplayProxyPacket type)
   return true;
 }
 
+// Modify the params that are used when getting the texture data for replay on the current driver.
+void ReplayProxy::ConvertProxyTextureIfNeeded(FetchTexture &tex, GetTextureDataParams &params)
+{
+  if(m_Proxy->IsTextureFormatSupported(tex.format))
+    return;
+
+  tex.format.special = false;
+  tex.format.compCount = 4;
+  tex.format.compByteWidth = 1;
+  tex.format.compType = eCompType_UNorm;
+
+  params.forceRGBA8unorm = true;
+  params.whitePoint = 1.0f;
```

Comment:
This looks good now - the only thing is that now all 'special' formats are being mapped silently to RGBA8. As I mentioned I want to only explicitly handle the ones you're currently implementing - ETC/EAC/ASTC. Formats like e.g. BC6 or R9G9B9E5 will need to remap to something different. An error message for those cases is enough for now.

I'm going to merge this PR in now rather than go for another round of changes, I'll fix this up in a post-merge commit.